apm-agent-core/src/test/resources/elasticapm.properties

```diff
@@ -5,3 +5,6 @@ disable_instrumentations=
 # in unit tests, the agent is loaded by the system class loader so we can't instrument bootstrap classes
 classes_excluded_from_instrumentation=java.*,com.sun.*,sun.*
 application_packages=co.elastic.apm
+
+# circuit breaker should be disabled by default, otherwise it is triggered while tests execute
+circuit_breaker_enabled=false
```

Comment:
The circuit breaker is off by default (except from the servlet container integration tests). It is enabled through a mock at the beginning of each `CircuitBreakerTest` test (which uses an isolated tracer and config).

Reply:
Good point, I've reverted this change in latest version of the PR.